src/git/config.go

```diff
@@ -26,6 +32,14 @@ const (
 
 	// apiURL is the url at which all APIs of github are rooted
 	apiURL = "https://api.github.com"
+
+	// root path of pre-commit files
+	BASE_FILES_PATH   string = "../../precommit/"
+	// pre-commit config file
+	PRE_COMMIT_CONFIG string = "../../precommit/CI/pre-commit-config.yaml"
```

Comment:
Let's keep the style consistent, maybe change it to camel case as is with the other variables?